closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -901,6 +902,23 @@ JNIEXPORT jlong JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_DigestCreateCon
     context->ctx = ctx;
     context->digestAlg = digestAlg;
 
+    // Create a second initalized openssl digest context. This is being done for performance reasons since
+    // creating and or re-initializing digest contexts later during processing is found to be expensive.
+    // This second context, context->cachedInitalizedDigestContext, will be copied over the working context,
+    // context->ctx, using the EVP_MD_CTX_copy_ex API whenever we wish to re-initalize this cipher. This occurs
+    // during an explicit reset of the cipher or whenever a final digest is computed.
```

Comment:
Updated comment styles to use /* and */ in this location and others where comments were added.

Reply:
Changed the spelling on this line and in a few other places where I misspelled initialized. Including the variable name `context->cachedInitalizedDigestContext` which was renamed to `context->cachedInitializedDigestContext`

Reply:
Our convention is to use ` * ` at the beginning of internal lines:
```c
    /*
     * Create a second initialized openssl digest context. This is being done for performance reasons since
     * creating and or re-initializing digest contexts later during processing is found to be expensive.
     * This second context, context->cachedInitializedDigestContext, will be copied over the working context,
     * context->ctx, using the EVP_MD_CTX_copy_ex API whenever we wish to re-initalize this cipher. This occurs
     * during an explicit reset of the cipher or whenever a final digest is computed.
     */
```
(See also the function comment.)

Reply:
Updated comments to include `*` at the start of each comment line.